editor/animation_track_editor.cpp

```diff
@@ -606,7 +606,6 @@ class AnimationTrackKeyEdit : public Object {
 			} break;
 			case Animation::TYPE_METHOD: {
 				p_list->push_back(PropertyInfo(Variant::STRING_NAME, "name"));
-				static_assert(VARIANT_ARG_MAX == 8, "PROPERTY_HINT_RANGE needs to be updated if VARIANT_ARG_MAX != 8");
 				p_list->push_back(PropertyInfo(Variant::INT, "arg_count", PROPERTY_HINT_RANGE, "0,8,1"));
```

Comment:
This still hardcodes `8` as max in the hint range, should likely change too. If it can take any number of args, it could be unbounded now (though should still be positive).

Same below.